webpack.config.js

```diff
@@ -19,8 +20,17 @@ module.exports = {
       }
     ]
   },
+  plugins: [new webpack.ProvidePlugin({ Buffer: ['buffer', 'Buffer'] })],
```

Comment:
We also need this to avoid the weird crash on old iPhones:


```js
  optimization: {
    minimizer: [new TerserPlugin({ terserOptions: { safari10: true } })]
  },
```

Reply:
Added and checked that it still works.